src/pages/mobile-app-development/cordova-ios-application-development-setup-to-deployment/index.md

```diff
@@ -0,0 +1,235 @@
+
+After selection the error should be gone, and it should show Team as the Team Name in Apple eveloper account and Signing Certificate Name.
+
+Do the same thing for Signing (Release) section, however in the file selection dialog slect the Ad Hoc distribution Profile.
+
+Now that the Code signing steps are done we either 
+* run the app directly on device
+* run the app on a simulator
+* generate an ipa file for distribution
+* upload app to appstore
+
+## Running the app directly on device
+
+To run the app on device connect the device to the Mac via USB,
+then in the top left corner in the list of devices select the cnnected device, and click the run or play button (black triangular button)
+<img src="https://image.ibb.co/k4xo15/Screen_Shot_2017_09_18_at_5_34_14_PM.png" alt="run device" width="100%">
+<img src="https://image.ibb.co/hjzhuQ/Screen_Shot_2017_09_18_at_5_36_55_PM.png" alt="run device" width="100%">
+
+The build status will be displayed in the status bar on the top of the window.
+If all goes fine, the app should be installed on the device, and automatically load in a while.
+
+## Running the app on a simulator
+
+The steps are same as running on device, but instead of an actual device we use the available iPhone and iPad simulators from the device list.
+
+## Generate an ipa file for distribution
+
+This approach can be done in case you require to distribute the app to the testing team, etc. However the device used by them must have a UDID present in the provisioning profile.
+
+From the Xcode menu select ```Product``` -> ```Clean```,
+Then ```Product``` -> ```Archive```,
+The Archives organizer appears and displays the new archive.
+<img src="https://image.ibb.co/iunfMG/6_ios_archive_organizer_2x.png" alt="ios archive organizer" width="100%">
+In the righthand side panel select Export option, al ist of optios will appear.
+
+To distribute your app to users with designated devices, select “Save for Ad Hoc Deployment.” The app will be code signed with the distribution certificate.
+
+To distribute your app for internal testing, select “Save for Development Deployment.” The app will be code signed with your development certificate.
+<img src="https://image.ibb.co/jQJLMG/6_ios_createappstorepackage_1_2x.png" alt="ios archive organizer export as ad hoc" width="100%">
+
+In the dialog that appears, choose a team from the pop-up menu and click Choose.
+<img src="https://image.ibb.co/gH2VMG/6_ios_export_choose_team_2x.png" alt="ios export select team"  width="100%">
+
+Next the device selection dialog popus up
+Select either *All devices* or *specific devices* click next.
+
+Next the review dialog is displayed,
+Here it will show the signing certificate and provisioning profile used for generating the build.
+Review and click next.
+Finally the file  save as popup is displayed to select the locationin the filesystem to store the exported app file.
+
+The app is exported as ``.ipa``` file .
```

Comment:
Is this correct?

Reply:
if you mean the typo : 'locationin' then i have fixed it, thank you.